README.md

```diff
@@ -60,7 +62,106 @@ int axis = -1;
 KokkosFFT::rfft(execution_space(), x, x_hat, KokkosFFT::FFT_Normalization::BACKWARD, axis); // FFT along -1 axis and batched along 0th axis
 ```
 
-## Building KokkosFFT
+In this example, the 1D batched ```rfft``` over 2D View along ```axis -1``` is executed. Some basic examples are found in [examples](https://github.com/CExA-project/kokkos-fft/tree/main/examples).
+
+## Disclaimer
+KokkosFFT is under development and subject to change without warning. The authors do not guarantee that this code runs correctly in all the environments.
+
+## Using KokkosFFT
+For the moment, there are two ways to use KokkosFFT: including as a subdirectory in CMake project or installing as a library. First of all, you need to clone this repo.
+```bash
+git clone --recursive https://github.com/CExA-project/kokkos-fft.git
+```
 
 ### CMake
-Up to now, we just rely on the CMake options for Kokkos.
+Since KokkosFFT is a header-only library, it is enough to simply add as a subdirectory. It is assumed that kokkos and kokkosFFT are placed under ```<project_directory>/tpls```.
+Here is an example to use KokkosFFT in the following CMake project. 
```

Comment:
```suggestion
Since KokkosFFT is a header-only library, it is enough to simply add as a subdirectory. It is assumed that kokkos and kokkosFFT are placed under `<project_directory>/tpls`.

Here is an example to use KokkosFFT in the following CMake project. 

```